lblod/frontend-mow-registry
# Update ember to version 4.12 lts

## Overview
This PR is responsible for two things:
- An upgrade to ember-source and ember-data to 4.12 lts (and their related packages)
- Dropping the `ember-cli-typescript` dependency and move to `ember-cli-babel` for typescript transpilation (see https://github.com/typed-ember/ember-cli-typescript/issues/1568)

##### connected issues and PRs:
None

### How to test/reproduce
- Go through the application and check whether you notice any (new) errors, especially related to ember-data.

### Checks PR readiness
